### PR TITLE
fix(copy): fix files and events copy

### PR DESCRIPTION
**Pull request by sceniclife**
This is the fix for the function parameters for files and events

### CODE REVIEW

**Comment by cognite-cicd** [comment]
#### Checkmarx scan completed with the following findings
|Lines|Severity|Category|File|Link|
---|---|---|---|---
[84](https://github.com/cognitedata/cognite-replicator/blob/fix-files-events-copy/cognite/replicator/__main__.py#L84) [73](https://github.com/cognitedata/cognite-replicator/blob/fix-files-events-copy/cognite/replicator/__main__.py#L73) |Medium|Path_Traversal|cognite/replicator/__main__.py|[Checkmarx](https://cognite.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1024667&projectid=145&pathid=1)

**Comment by gaetan-h** [comment]
thanks, we have lots of errors from mypy, they appeared in master, but did not show in the previous github actions, not sure how to fix it. But looks good otherwise.

**Review by gaetan-h — APPROVED** [review]
LGTM

**Comment by gaetan-h** [comment]
You may also want to change the version if you want it to appear in pypi

**Comment by sceniclife** [comment]
> You may also want to change the version if you want it to appear in pypi

I'll do that now.

File: cognite/replicator/events.py [hunk 1]
```diff
@@ -84,6 +84,7 @@ def copy_events(
     client: CogniteClient,
     src_filter: List[Event],
     jobs: queue.Queue = None,
+    exclude_fields: Optional[List[str]] = None,
```
Comment (petreeb) [inline cognite/replicator/events.py:87]: Can you add the new parameter to the comments

File: cognite/replicator/files.py [hunk 2]
```diff
@@ -76,6 +76,7 @@ def copy_files(
     client: CogniteClient,
     src_filter: List[FileMetadata],
     jobs: queue.Queue = None,
+    exclude_fields: Optional[List[str]] = None,
```
Comment (petreeb) [inline cognite/replicator/files.py:79]: same as above

**Review by petreeb — APPROVED** [review]

**Comment by sceniclife** [comment]
Yeah the mypy stuff is coming from somewhere. Not sure.

**Pull request merged**